components/style/values/specified/text.rs

```diff
@@ -349,3 +351,108 @@ impl Parse for TextDecorationLine {
         }
     }
 }
+
+/// Specified value of text-align property.
+#[cfg_attr(feature = "gecko", derive(MallocSizeOf))]
+#[derive(Clone, Copy, Debug, Eq, Hash, PartialEq)]
+pub enum TextAlign {
+    /// Keyword value of text-align property.
+    Keyword(ComputedTextAlign),
```

Comment:
Given that `ComputedTextAlign` is used in the specified value I'd prefer if it were in this file as `TextAlignKeyword` and `computed::TextAlign` is an alias